src/test/github-non-pipeline-cases.spec.ts

```diff
@@ -51,7 +51,7 @@ const commandsDataProvider: CommandDataProviderItem[] = [
     commandLine: "bot hrlp", // intentional typo
     expected: {
       startMessage:
-        '@somedev123 Unknown command "hrlp"; Available ones are bench-all, bench-vm, bench, fmt, merge, rebase, sample, try-runtime. Refer to [help docs](http://localhost:3000/static/docs/latest.html) and/or [source code](https://github.com/paritytech/command-bot-scripts).',
+        '@somedev123 Unknown command "hrlp"; Available ones are bench-all, bench-bm, bench, fmt, merge, rebase, sample, try-runtime, update-ui. Refer to [help docs](http://localhost:3000/static/docs/latest.html) and/or [source code](https://github.com/paritytech/command-bot-scripts).',
```

Comment:
How come you had to change this test in this PR?

Reply:
I believe command-bot downloads `command-bot-scripts` anew.  
So, updates in the scripts lead to needing updates in tests: https://github.com/paritytech/command-bot-scripts/pull/39/files. 

@mordamax is this the behaviour we want?